app/README.md

```diff
@@ -37,3 +42,60 @@ npm run start:app
 ```
 
 You can open the Measure Repository Service frontend application by navigating to [http://localhost:3001](http://localhost:3001) in your browser.
+
+
+# Frontend Features
+
+## Navigation
+
+The application is divided into two main sets of functionality, _Repository_ and _Authoring_, both of which are accessible from tabs at the top of the page.
+
+<img src="./static/tabs.png" alt="Screenshot of tabs" width="200"/>
+
+The Repository tab provides functionality for access and interacting with measure artifacts that are in the FHIR Measure Repository Service. The Authoring tab focuses on measure artifacts that are in the process of being drafted. Artifacts can be copied or moved between these two spaces through _draft_ and _release_ actions, described in the [Viewing](#viewing) and [Editing](#editing) sections.
```

Comment:
```suggestion
The Repository tab provides functionality for access and interacting with measure and library artifacts that are in the FHIR Measure Repository Service. The Authoring tab focuses on measure and library artifacts that are in the process of being drafted. Artifacts can be copied or moved between these two spaces through _draft_ and _release_ actions, described in the [Viewing](#viewing) and [Editing](#editing) sections.
```
I would potentially rephrase "Artifacts can be copied or moved between..." because technically they are not being copied/moved between the two because their version/status changes between the two- is this too nitpicky? You could instead keep it simple to something like "Artifacts can be drafted or released beween these two spaces..." 

I would also maybe reword "The Authoring tab focuses on measure artifacts that are in the process of being drafted" since they are drafts...maybe just say "focuses on draft measure and library artifacts" or something else that is perhaps clearer on the fact that they are drafts.

Reply:
So the problem is that there can technically be drafts in the FHIR repo as well, so the app is kind of a temporary space for ... lets maybe say "unofficial" draft measures (because they're not in the official repo yet). I'm trying to differentiate the purpose of the authoring space as being more temporary and in progress.

Reply:
```suggestion
The Repository tab provides functionality for access and interacting with artifacts that are in the FHIR Measure Repository Service. The Authoring tab focuses on artifacts that are in the process of being drafted. Artifact content can be copied or moved between these two spaces through _draft_ and _release_ actions, described in the [Viewing](#viewing) and [Editing](#editing) sections.
```
What do you think about changing it to "Artifact content" so that we can show that the main content is being copied or moved, but this implies that there may be some more meta-data type fields that may change in that process?